### ISSUE TITLE
Always show package:flutter matches before dart:ui matches on autocomplete

### ISSUE THREAD

**Issue by jacob314** [flutter/flutter-intellij#2722]
See related docs.flutter.io issue for more details.
https://github.com/flutter/flutter/issues/23143

**Comment by Solido** [comment]
It would make sense to also apply this to cupertino package that show on top of material.
Thank you.

**Comment by scheglov** [comment]
Do you really mean "autocomplete", or something else, e.g. `Import Library` quick fix?
If it the first, please provide an example.

If it is the second, yeah, I can see why you think that it could be useful.
![image](https://user-images.githubusercontent.com/384794/48286634-21f8bf00-e41b-11e8-8a6c-01ac39234de8.png)

It is not `100%` clear decision, though. Probably for most of the users it is, because they build their UIs using components, so of course they want `package:flutter:material.dart`. But what if you really work on lower level stuff, like implementing some custom widget, and prefer `painting.dart`? If we could remove some suggestions at all, this probably would be OK. But reordering them in non-obvious ways, or with non-obvious rules is IMHO bad UX.

**Comment by Solido** [comment]
In my case each time I use live template to create my widget then I got the message that the import are missing so I use the keyboard to get the suggestions.

At this step I need to use the down key to get to material which is the best solution 95% of the time.
I'm sure beginner said ok to cupertino because they don't understand the difference ... and guess what ? they import cupertino on all their basics widget because the IDE suggested it which the worse UX case.

I confess I still have some files to cleanup ;)

**Comment by jacob314** [comment]
I agree that material, cupertino, then dart:ui is the right priority order for users.
Very few developers should be using the version from dart:ui directly. Potentially in the future we could apply some simple bayesian logic to determine the priority order of the import based on the probability of the import being used for a particular class globally across all of github and locally in the user's project.
